src/Serilog.Sinks.Console/Serilog.Sinks.Console.csproj

```
@@ -4,7 +4,7 @@
     <Description>A Serilog sink that writes log events to the console/terminal.</Description>
     <VersionPrefix>4.2.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;net5.0</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;net5.0;net6.0;net7.0</TargetFrameworks>
```

per the skeleton policy in the comment, I would not add net7.0 (and would also not add `net8.0` in a few months either)
```suggestion
    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;net5.0;net6.0</TargetFrameworks>
```

---

FWIW I think signaling support of .Net8.0 now does make sense. I just submitted a PR yesterday for another project that makes Grpc work with named pipes, that already had .Net8.0.  Seems to be in the mix. Just be aware that some folks may not have the SDK on their box for whatever reason and some may not be able to add it yet for IT policy reasons.

---

If we have a VERSIONING.md, we can point people to that.
It will mention that the default assumption is that the world uses Serilog and if it is not happy on the latest FW release or preview, that's a bug that needs fixing immediately, with a link to the download counts so people can stop relying on inferring support based on the presence of a variant that *requires* a TFM
It will also mention that just because there is only a net6.0 target in the package does not mean that it can't be used with 8/9/10/11/12 etc

For me the important part of supporting net8.0 is to have the test suite use it

This means two things
- people forking the repo need it on their box
- the CI needs to support it (if its -latest images, that's covered)
For gRPC stuff, it would not surprise me if they have a dependency on new stuff in 7.0 or 8.0

In that case, they will likely no longer support 6.0, which is a breaking change which we would signal with a major version bump
If Serilog added a net8.0 target for a reason (some newer API means we can implement a feature or do something quicker or with less memory), then the CI would need a new SDK, and the package would get a new TFM - but there is zero impact on devs using the package - they can upgrade and a net6 SDK will safely ignore the new package (and use the most relevant impl that's already in the package)

cc @nblumhardt @SimonCropp

---

Sorry, realised a point you were making about 'IT policy reasons'; IMO:

- `main` should never use non-RTM TFMs (i.e. baseline stance is no point releasing a final version that relies on a thing in preview)
- `dev` and PRs can use previews if developing a new feature or tuning that requires preview - the critical bit is the CI env having them; we would not merge without that anyway
- because CI will cover the preview SDK requirement, people contributing PRs can do a local edit to temporarily e.g. skip testing net8.0